Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Services/ApiGatewayRouteConfigService.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections;
+using System.Text.Json;
+using Amazon.Lambda.TestTool.Models;
+using Amazon.Lambda.TestTool.Services.IO;
+using Microsoft.AspNetCore.Routing.Template;
+
+namespace Amazon.Lambda.TestTool.Services;
+
+/// <inheritdoc cref="IApiGatewayRouteConfigService"/>
+public class ApiGatewayRouteConfigService : IApiGatewayRouteConfigService
```

Comment:
can you add test cases for this class? and any other ones that youve added

Reply:
Done. I added wrappers for Environment and Directory. I am not going to add tests for those. They are 1:1 mapping of the system classes.